### PR TITLE
Adding php example and the travis configration

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+branches:
+  only:
+    - master
+
+language: php
+php:
+  - '7.1'
```
Comment (bshaffer) [inline .travis.yml]: nit: why single quotes?

File: README.md [hunk 2]
```diff
@@ -3,6 +3,13 @@
 This is a demo project for showing how to run unit tests with Cloud
 Container Builder.
 
+## Test Runners
+
+Currently we only have a test runner for phpunit at `php/test/runner`
+directory. We built it and push it to
+gcr.io/cloud-dpes/phpunit-test-runner. This image is publicly
```
Comment (bshaffer) [inline README.md]: We should be able to make this repository visible to everyone. Currently, following that link results in "You don't have permission to view Container Registry".

File: README.md [hunk 3]
```diff
@@ -3,6 +3,13 @@
 This is a demo project for showing how to run unit tests with Cloud
 Container Builder.
 
+## Test Runners
+
+Currently we only have a test runner for phpunit at `php/test/runner`
+directory. We built it and push it to
```
Comment (bshaffer) [inline README.md]: Should be `We build it and push it`

File: php/test/runner/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Dockerfile for running phpunit within a cloudbuild step.
+
+FROM python:2.7-alpine
```
Comment (bshaffer) [inline php/test/runner/Dockerfile]: I think it would make more sense to use a php base image and install python.

File: .travis.yml (hunk 1 above)
Comment (tmatsuo) [inline .travis.yml]: no reason, but it works

File: README.md (hunk 3 above)
Comment (tmatsuo) [inline README.md]: Changed to `We already built and pushed it`

File: README.md (hunk 2 above)
Comment (tmatsuo) [inline README.md]: It is already public. It is not supposed to work via `http`

File: php/test/runner/Dockerfile (hunk 4 above)
Comment (tmatsuo) [inline php/test/runner/Dockerfile]: Ok, I removed gcloud, and just use alpine for the base now

**Comment by tmatsuo** [comment]
@bshaffer Thanks, merging

**Pull request merged**